test/functional/interface_rest.py

```diff
@@ -235,13 +234,13 @@ def run_test(self):
         response_hex = self.test_rest_request("/block/{}".format(bb_hash), req_type=ReqType.HEX, ret_type=RetType.OBJ)
         assert_greater_than(int(response_hex.getheader('content-length')), BLOCK_HEADER_SIZE*2)
         response_hex_bytes = response_hex.read().strip(b'\n')
-        assert_equal(binascii.hexlify(response_bytes), response_hex_bytes)
+        assert_equal(response_bytes.hex().encode(), response_hex_bytes)
```

Comment:
nit: Possible alternative, though I think both variants are fine:
```suggestion
        assert_equal(response_bytes.hex(), response_hex_bytes.decode())
```

Reply:
Yeah, I didn't want to disrupt the flow; it looked like the asserts in the file were of the kind `assert_equal(actual, desired)` (for the most part), i.e. the left arg being compared to the right.

If that doesn't matter much I'll just change it.